### PR TITLE
2019 workshop

### PR DESCRIPTION
**Pull request by daniellindau**
Simplify code
Update exercises
python 3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,39 +3,20 @@ This is a OpenID Connect example, used as an assigment in the OAuth2 and OpenID
...
-### app.py
-4. Add the token to the api request
-    You should now be able to request data from 
-
-5. Add openid to the requested scopes, in configuration.
-    You should get an id_token from the token response
-
-6. Validate the id_token
-    Use the validator to validate the id_token
+TODO!
```
Comment (anestos) [inline README.md:13]: todo what?

File: README.md [hunk 2]
```diff
@@ -48,18 +29,14 @@ Settings.json is used as a configuration file for the example app. Change the va
...
+`client_id`     | string  |    ✓      |          | The id for the client. Used to authenticate the client.
 `client_secret` | string  |    ✓      |          | The shared secret to use for authentication against the token endpoint.
-`discovery_url` | URL     |    ✓      |          | The URL where the metadata of the sever can be found. Should contain information about the endpoints and keys to be used. Configuration from the discovery url will override configuration from settings.json.
-`scope`         | string  |           | `openid` | The scopes to ask for.
-`jwks_uri`      | URL     | if `discovery_url` is not set and the `openid` scope is requested          |          | The URL that points to the JWK set.
-`authorization_endpoint` | URL | if `discovery_url` is not set     |          | The URL to the authorization_endpoint.
-`token_endpoint`| URL     |           |          | The URL to the token_endpoint. Mandatory if `discovery_url` is not set.
-`issuer`        | string  | if the `openid` scope is requested and `discovery_url` is not set          |          | The ID of the token issuer.
+`isser`         | URI     |    ✓      |          | The OAuth2 issuer
```
Comment (anestos) [inline README.md:34]: remove

File: README.md [hunk 3]
```diff
@@ -48,18 +29,14 @@ Settings.json is used as a configuration file for the example app. Change the va
...
+`scope`         | string  |           |          | The scopes to ask for.
+`issuer`        | string  |    ✓      |          | The ID of the token issuer.
```
Comment (anestos) [inline README.md:36]: The OAuth/OpenID issuer?

File: templates/index.html [hunk 4]
```diff
@@ -120,47 +84,30 @@
     <div class="row">
         <div class="col-md-3"></div>
         <div class="col-md-5">
-            <h2 class="welcome">Welcome <span id="sessionname"></span>!</h2>
+            <h2 class="welcome">Welcome</span>!</h2>
```
Comment (anestos) [inline templates/index.html:87]: remove `</span>`

**Review by anestos — APPROVED** [review]
Some minor things, other than that looks good

File: README.md (hunk 1 above)
Comment (daniellindau) [reply README.md:13]: Add in the assignments. I removed the headline instead, since we will hand it out in the workshop.

**Pull request merged**